.travis.yml

```diff
@@ -7,11 +7,10 @@ script: bundle exec rake travis
 matrix:
   fast_finish: true
   include:
-    - rvm: 1.9.3
-    - rvm: 2.0.0
-    - rvm: 2.1.0
-    - rvm: 2.1.5
-    - rvm: 2.1.5
+    - rvm: 2.0
+    - rvm: 2.1
+    - rvm: 2.2
+    - rvm: 2.2
```

Comment:
I think it does. We're doing a separate build of our changes in `fog` against released versions of `fog-core` and `fog-json` and also against the master branches of those two gems.